fetchy/extractors/vaccinationDataPerDhbExtractor.ts

```diff
@@ -0,0 +1,27 @@
+import { DateTime } from "luxon";
+import { DhbVaccineDoseData } from "../../shared/types/VaccineDataTypes";
+import { extractDhbData_Including_and_Before_15_02_22 } from "./vaccinationDataPerDhbExtractors/extractorIncludingAndBefore15_02_22";
```

Comment:
:nit: Prefer not to have underscores in this name, we can do:

```suggestion
import { extractDhbDataIncludingAndBefore15Feb22 } from "./vaccinationDataPerDhbExtractors/extractDhbDataIncludingAndBefore15Feb22";
```